stream_alert_cli/terraform/kinesis_streams.py

```diff
@@ -29,6 +29,7 @@ def generate_kinesis_streams(cluster_name, cluster_dict, config):
     """
     prefix = config['global']['account']['prefix']
     config_modules = config['clusters'][cluster_name]['modules']
+    shard_level_metrics = config['global']['infrastructure']['monitoring']['shard_level_metrics']
```

Comment:
what happens if this doesn't exist in the config?

Reply:
`KeyError` exception I guess. I should default it to an empty list, if the value isn't there.